### ISSUE TITLE
[Feature Request] A project scafold or boilerplate of a Robyn API

### ISSUE THREAD

**Issue by juanfrilla** [sansyrox/robyn#360]
Hello, do anybody have a Robyn boilerplate to look what an API archives structures looks like?
Like this: https://github.com/tiangolo/full-stack-fastapi-postgresql
If not, i suggest you implement one.

**Comment by sansyrox** [comment]
Hey @juanfrilla , that's a great idea. Unfortunately, there is nothing like that at the moment. We will be happy to create one 😄 Is there something specific that you would like to see? (frontend framework, database provider, or something else)?

**Comment by juanfrilla** [comment]
@sansyrox Thanks. With the backend folder structures, some .py files inside of it and some code inside those .py files, its ok for me.  For example a simple CRUD API.
E.g database connection, some endpoints, some models, serialization.

About technologies:
Referred to databases, I don't care which one you use because whatever you put is changing one for another.
Referred to SysAdmin/Devops: It will be nice that the boilerplate will be ready for production, with some dockerfiles or docker-composes inside of it. 
If you want to add some CI/CD its valid for me as well, for example with Github Actions.
And to finish, if you consider, Kubernetes with microservices (this could be complicating it a little bit).


PD: I discover your framework today and it looks really promising and fast. It's curious that is a Python framework written in Rust.

**Comment by sansyrox** [comment]
Thank you for the detailed examples @juanfrilla 😄 

> I discover your framework today and it looks really promising and fast. It's curious that is a Python framework written in Rust.
Keep going 😄🙏

Thank you! 😄 I am glad you liked it ✨

**Comment by sansyrox** [comment]
Hey @Noborita9 👋 

Would you be interested in something like this?

**Comment by Noborita9** [comment]
Yeah, I am willling to take this. Should I create a repo on my own or where to let it there

**Comment by sansyrox** [comment]
Oh, perfect. I can create a Repo and add you as a collaborator there. Would that work for you?

**Comment by sansyrox** [comment]
@Noborita9 , I robyn will be a part of a larger organization and I would want the template to live there. So, it will be easier that way imo.

**Comment by Noborita9** [comment]
Yeah, seem great just send me the invite and I'll will be working there.

**Comment by sansyrox** [comment]
@Noborita9 , I have added you to the repo. Can you check if you have access?

**Comment by Noborita9** [comment]
@sansyrox I just made a push, so everything is fine for me to work on it!
This week I've been a bit busy, so I expect next week to be pushing some content there

**Comment by juanfrilla** [comment]
It would be nice if you the repo here guys 😉

**Comment by Noborita9** [comment]
Hi @juanfrilla, here's the [repo](https://github.com/sansyrox/robyn-starter), I stopped making it because I was interested at having routers, so when they are added ([They are on they way](https://github.com/sansyrox/robyn/issues/453))I will make a first version of it

**Comment by brock-afk** [comment]
Is this something that there is still interest in even with the robyn CLI --create option? If so, I would be interested in giving it a shot. I have an idea of the direction I would go with this, but I am open to guidance if there is a specific vision for a project like this.

**Comment by zweix123** [comment]
Can this question be extended to add a command-line sub command to robyn to give it scaffold like functionality?
